src/utils/cfgreader.py

```diff
@@ -79,6 +79,19 @@ def _do_read(self, rep: str) -> float:
         return float(rep)
 
 
+class FloatRangeVar(EVar):
+    """
+    An env value parsed as a FloatRange. Value of "CRATE_BACKOFF_FACTOR"
+    expected to be in range of 0 to 120 otherwise it will raise Value error.
+    """
+
+    def _do_read(self, rep: str) -> float:
+        backoff_factor = int(os.environ.get('CRATE_BACKOFF_FACTOR'))
+        if backoff_factor not in range(0,120):
+            raise ValueError('value out of range: {}'.format(backoff_factor))
+        return float(rep)
+
+
```

Comment:
Can we make this class more generic so it works for any range, not just the 0-120 range?
e.g.

```python
class FloatRangeVar(EVar):
   def __init__(var_name, default_value, mask_value, lo, hi):
    # super(var_name, default_value, mask_value)
    # ...check lo <= hi

   def _do_read(self, rep: str) -> float:
     value = float(rep)
     if value not in range(self.lo, self.hi):
       raise ValueError(...)
     return value
```